### PR TITLE
[16.0][mig] account_tax_group_widget_base_amount: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Comment by Gelo-fl** [comment]
LGTM https://www.awesomescreenshot.com/video/19024270?key=8ea582aac007d3070d8f2675cd79330c

**Review by Gelo-fl — APPROVED** [review]
LGTM https://www.awesomescreenshot.com/video/19024270?key=8ea582aac007d3070d8f2675cd79330c

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ addon | version | maintainers | summary
 [purchase_stock_picking_return_invoicing](purchase_stock_picking_return_invoicing/) | 16.0.1.0.0 | [![pedrobaeza](https://github.com/pedrobaeza.png?size=30px)](https://github.com/pedrobaeza) [![MiquelRForgeFlow](https://github.com/MiquelRForgeFlow.png?size=30px)](https://github.com/MiquelRForgeFlow) | Add an option to refund returned pickings
 [sale_order_invoicing_grouping_criteria](sale_order_invoicing_grouping_criteria/) | 16.0.1.0.0 | [![pedrobaeza](https://github.com/pedrobaeza.png?size=30px)](https://github.com/pedrobaeza) | Sales order invoicing grouping criteria
 [stock_picking_return_refund_option](stock_picking_return_refund_option/) | 16.0.1.0.0 | [![sergio-teruel](https://github.com/sergio-teruel.png?size=30px)](https://github.com/sergio-teruel) | Update the refund options in pickings
+[account_tax_group_widget_base_amount](account_tax_group_widget_base_amount/) | 16.0.1.0.0 |  | Adds base to tax group widget.
```
Comment (pedrobaeza) [inline README.md]: Please don't change manually this file.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**